tests/alembic_migrations/versions/v_8c05da87cb79_add_hx_tkg_to_obs_raw/test_on_real_tables/test_on_real_tables.py

```diff
@@ -0,0 +1,75 @@
+"""
+These tests simulate applying history tracking to metadata tables containing
+data, as we would find in a real migration. We do this by populating the tables
+with data, applying the migration, and then examining the resulting history
+tables.
+"""
```

Comment:
So it seems like you test the successful creation of the obs history table, but not its behaviour as the obs_raw table is subsequently updated. Do I recall correctly that a previous PR included tests of that fuctionality (not on obs_raw, but on some other table?)

Reply:
Yes, that's correct. There's a comment buried in the body of the test code that says as much. I'll move it up here. Thx.

Reply:
But on further consideration, the test ought to include some kind of verification that a history record is written when an insert, update, and delete is performed. Thanks for prompting further thought!